src/tests/system/tests/test_authentication.py

```diff
@@ -48,6 +48,144 @@ def test_authentication__with_default_settings(
     ), "User logged in with an invalid password!"
 
 
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.importance("critical")
+def test_authentication__change_password(
+    client: Client,
+    provider: GenericProvider,
+):
+    """
+    :title: User logins and issues a password change
+    :setup:
+        1. Create user 'user'
+        2. Start SSSD
+    :steps:
+        1. Login as user
+        2. Issue password change and enter a bad confirmation password
+        3. Issue password change and enter a good confirmation password
+        4. Login with old password
+        5. Login with new password
+    :expectedresults:
+        1. User is authenticated
+        2. Password is change is unsuccessful
+        3. Password change  is successful
```

Comment:
extra whitespace?  Not a big deal since I barely noticed it.   Kind of surprised one of the linters didn't complain.

Reply:
It's in the comment block, fixed.